guides/v2.3/extension-dev-guide/declarative-schema/index.md

```diff
@@ -40,3 +40,5 @@ Once a module is converted to the declarative schema approach, it cannot be reve
   * Performing data migration with inside DDL operations
   * Renaming tables, columns, and other entities
   * Adding partitions and options to a table
+
+* *Revertable schema patch* - A `schema patch` that contains a `revert()` method with provided revertable functionality.
```

Comment:
My previous comment about revertable changes is fair for this conclusion as well.

Reply:
**fixed**

Could you please verify a new fix?

Thank you!